README.md

```diff
@@ -49,6 +49,7 @@ This README contains some high level overview information on Atomic App. The det
 8. [Specification coverage](docs/spec_coverage.md)
```

Comment:
hence the reaosn for spec_coverage.md doc.

Yeah, we should rip those out... but that's a spec change that should be put in a separate PR.